group_vars/all/vlans.yaml

```diff
@@ -1,17 +1,326 @@
+    switches:
+      - OCT8-SW-MGMT
+  950:
+    name: "UMA-NESE"
+    description: "NESE BGP VLAN for UMass"
+    switches:
+  1000:
+    name: "CHAMELEON-INTERNAL"
+    description: "Chameleon Internal Network"
+    switches:
+      - OCT8-SW-TORS
+  1001:
+    name: "CHAMELEON-NEUTRON"
+    description: "Chameleon neutron provisioning"
+    switches:
+      - OCT8-SW-TORS
+  1004:
+    name: "KUMO-INTRANET"
+    description: "Intranet (routable to internet). SNMP for client nodes."
+    switches:
+      - oct_data
+  2470:
+    name: "NERC-FLOATING-201"
+    description: "*eth4: floating ip (vlan 201) - tenant public IPs"
+    switches:
+  2471:
+    name: "NERC-EXTERNAL-200"
+    description: "*eth3: external network (vlan 200) - public API / endpoints"
+    switches:
+  2472:
+    name: "NERC-INTERNAL-101"
+    description: "eth0.101: internal api (vlan 101) - internal to node"
+    switches:
+  2473:
+    name: "NERC-PRIVATE-102"
+    description: "eth0.102: tenant private (vlan 102) - VMs sit here, private network/VXLAN"
+    switches:
+  2476:
+    name: "NERC-ADMIN-100"
+    description: "NERC-Admin network 1->eth0:provisioning network(vlan 100)-deploying images/DHCP> PAT out"
+    switches:
+  2477:
+    name: "NERC-OBM"
+    description: "NERC - OBM/MGMT Network 1 -> eth1: management/ipmi - undercloud to DRAC/BMC 10.255"
+    switches:
+  2478:
+    name: "NERC-STORAGE-103"
+    description: "eth2: storage (vlan 103) - ceph/jumboframe? 10.255"
+    switches:
+  3100:
+    name: "AL2S-3100"
+    description: "AL2S VLAN 3100 for Fabric"
+    switches:
+      - oct_hub
+  3101:
+    name: "AL2S-3101"
+    description: "AL2S VLAN 3101 for Fabric"
+    switches:
+      - oct_hub
+  3102:
+    name: "AL2S-3102"
+    description: "AL2S VLAN 3102 for Fabric"
+    switches:
+      - oct_hub
+  3103:
+    name: "AL2S-3103"
+    description: "AL2S VLAN 3103 for Fabric"
+    switches:
+      - oct_hub
+  3104:
+    name: "AL2S-3104"
+    description: "AL2S VLAN 3104 for Fabric"
+    switches:
+      - oct_hub
+  3105:
+    name: "AL2S-3105"
+    description: "AL2S VLAN 3105 for Fabric"
+    switches:
+      - oct_hub
+  3106:
+    name: "AL2S-3106"
+    description: "AL2S VLAN 3106 for Fabric"
+    switches:
+      - oct_hub
+  3107:
+    name: "AL2S-3107"
+    description: "AL2S VLAN 3107 for Fabric"
+    switches:
+      - oct_hub
+  3108:
+    name: "AL2S-3108"
+    description: "AL2S VLAN 3108 for Fabric"
+    switches:
+      - oct_hub
+  3109:
+    name: "AL2S-3109"
+    description: "AL2S VLAN 3109 for Fabric"
+    switches:
+      - oct_hub
+  3801:
+    name: "CSAIL-3801"
+    description: "CSAIL - ESI Floating IP"
+    switches:
+      - oct_data
+  3802:
+    name: "CSAIL-3802"
+    description: "CSAIL Floating IP 2"
+    switches:
+      - oct_data
+  3803:
+    name: "CSAIL-3803"
+    description: "CSAIL MaaS"
+    switches:
+      - oct_data
+  3808:
+    name: "****"
+    description: "Public IPs****"
```

Comment:
I don't think we have 3808 in use anywhere. 

There's one network with the ID 3800 which is called "MEETME_FLAT_NETWORK" on the cisco switches, that somehow is tied to the other CSAIL networks (3801 and 3802, CSAIL mentioned they see traffic on that network) and I don't understand if it's safe to disable that or not so I haven't. It was setup before me or even Rado joined MOC.

```
MOC-NEU-PODB-R3C19-1# show vlan id 3800

VLAN Name                             Status    Ports
---- -------------------------------- --------- -------------------------------
3800 MEETME_FLAT_NETWORK              active    Po25, Po26, Eth1/21, Eth1/27
                                                Eth1/28, Eth1/41, Eth1/42
                                                Eth1/43, Eth1/44, Eth2/3, Eth2/5
                                                Eth2/6
```

Reply:
I'll put it in the file but won't assign it to any switch for now - it's not on any of the OCT switches currently anyway.